packages/mantine/src/components/code-editor/CodeEditor.tsx

```diff
@@ -143,6 +141,10 @@ export const CodeEditor: FunctionComponent<CodeEditorProps> = (props) => {
         }
     };
 
+    const [hasMonacoError, setHasMonacoError] = useState(false);
+    const renderErrorOutline = useMemo(() => !!error || hasMonacoError, [hasMonacoError, error]);
+    const {classes, theme} = useStyles({error: renderErrorOutline});
```

Comment:
While at it we should probably make it overridable via the theme.

```suggestion
    const {classes, theme} = useStyles({error: renderErrorOutline}, {name: 'CodeEditor'});
```